frontend/src/metabase/visualizations/visualizations/Map.jsx

```diff
@@ -237,10 +237,12 @@ export default class Map extends Component {
         return null;
       },
       getProps: () => ({
-        options: Object.entries(MetabaseSettings.get("custom-geojson", {})).map(
-          // $FlowFixMe:
-          ([key, value]) => ({ name: value.name, value: key }),
-        ),
+        options: _.chain(
+          Object.entries(MetabaseSettings.get("custom-geojson", {})),
+        )
+          .map(([key, value]) => ({ name: value.name, value: key }))
+          .sortBy(x => x.name.toLowerCase())
```

Comment:
This won't handle strings with accents correctly, but that that's probably ok for now. Browsers have `localeCompare` to handle that, but it doesn't work with `_.sortBy`.

Reply:
Oh also, I think we do validate `name` in the UI, but I'm not sure if there's also backend validation. Adding `(name || "")` doesn't hurt since this will break if someone has some old map stored will a null name.